recovery/translation_ko.ts

```diff
@@ -872,7 +873,7 @@ Please provide it locally or connect to the internet.</source>
 See debug log for details.</source>
         <oldsource>OS(es) Downloaded with errors.
 See debug log for details</oldsource>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
Empty translation here - was that deliberate?

Reply:
Oh, what the... thanks for your check. I will change it now